packages/flutter/lib/src/painting/debug.dart

```diff
@@ -157,11 +157,13 @@ PaintImageCallback? debugOnPaintImage;
 /// This has no effect unless asserts are enabled.
 bool debugInvertOversizedImages = false;
 
+const int _imageOverheadAllowanceDefault = 128 * 1024;
+
 /// The number of bytes an image must use before it triggers inversion when
 /// [debugInvertOversizedImages] is true.
 ///
-/// Default is 1024 (1kb).
-int debugImageOverheadAllowance = 1024;
+/// Default is 128kb.
+int debugImageOverheadAllowance = _imageOverheadAllowanceDefault;
```

Comment:
with this change, should we just default the value to on?

Reply:
My sense is that that might be unexpected w/o a user opt-in?

I think this is a nice, lightweight feature (a debugger toggle to highlight oversize images), but personally think that having an API to query for information about active images, and tooling in DevTools to display those images, would be the next avenue to explore.